atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -589,6 +590,9 @@ private void refreshTokenRanges() {
                     if (numTries >= MAX_TRIES_SAME_HOST) {
                         shouldRetryOnDifferentHost = true;
                     }
+                } else if (isFastFailoverException(e)) {
```

Comment:
this doesn't look like new code... was it reinstated after a merge or something?